ImputationPipeline/CreateAggregationSets/CreateAggregationSets.py

```diff
@@ -44,7 +43,7 @@ def __init__(self, workspace_namespace, workspace_name):
         entity_types_response = fapi.list_entity_types(self.workspace_namespace, self.workspace_name)
```

Comment:
You can now get rid of the `pre_existing_aggregation_set` and `next_aggregation_set` methods